test-data/unit/stubgen.test

```diff
@@ -2580,3 +2580,18 @@ class A:
 def f(x: int, y: int) -> int: ...
 @t.overload
 def f(x: t.Tuple[int, int]) -> int: ...
+
+
+[case testProtocol_semanal]
+from typing import Protocol
+
+class P(Protocol):
```

Comment:
Let's also add a generic `Protocol[T]` class here 🤔

Reply:
You mean like this?
```python
T = TypeVar('T')
class PT(Protocol[T]):
    def f(self, x: T) -> T:
        ...
```
That results in:
```python
  T = TypeVar('T')

  class PT(Protocol):
      def f(self, x: T) -> T: ...
```
So I guess that needs to be handled explicitly in my implementation?

Reply:
Yes 🙂

Reply:
Done ✅